src/components/screens/passphraseBackup/index.js

```diff
@@ -1,72 +1,77 @@
+import { deviceWidth, deviceHeight, SCREEN_HEIGHTS } from 'utilities/device';
+import { PrimaryButton } from 'components/shared/toolBox/button';
 import PassphraseCopy from 'components/shared/passphraseCopy';
 import withTheme from 'components/shared/withTheme';
 import { P, A } from 'components/shared/toolBox/typography';
+import HeaderBackButton from 'components/navigation/headerBackButton';
 import getStyles from './styles';
 
 const isSmallScreen = deviceHeight() < SCREEN_HEIGHTS.MD;
 const qrCodeSize = deviceWidth() * (isSmallScreen ? 0.64 : 0.72);
 
-@connect(state => ({
-  passphrase: state.accounts.passphrase,
-}))
-class PassphraseBackup extends React.Component {
-  state = {
-    passphraseRevealed: false,
-  };
+const PassphraseBackup = ({
+  styles,
+  t,
+  theme,
+  navigation,
+  sharedData: data,
+  nextStep,
+}) => {
+  const [passphraseRevealed, setPassphraseReveal] = useState(false);
 
-  showQRCode = () => {
-    this.setState({ passphraseRevealed: !this.state.passphraseRevealed });
+  const toggleQRCode = () => {
+    setPassphraseReveal(!passphraseRevealed);
   };
 
-  render() {
-    const {
-      styles, passphrase, theme, t
-    } = this.props;
-    const { passphraseRevealed } = this.state;
+  const nextScreen = () => {
+    nextStep({ passphrase: data.recoveryPhrase });
+  };
 
-    return (
-      <SafeAreaView style={[styles.wrapper, styles.theme.wrapper]} >
-        <ScrollView style={styles.container} >
-          <PassphraseCopy passphrase={passphrase} />
-          <P style={[styles.QRText, styles.theme.text]}>
-            {t('Private use only')}
-            <A style={styles.button} onPress={this.showQRCode}>
-              &nbsp;{this.state.passphraseRevealed ? t('Hide QR code') : t('Show QR code')}
-            </A>
-          </P>
-          {passphraseRevealed && (
-            <View style={styles.qrCodeContainer}>
-              <QRCode
-                value={passphrase}
-                size={qrCodeSize}
-                color={
-                  theme === themes.light
-                    ? colors.light.black
-                    : colors.dark.white
-                }
-                backgroundColor={
-                  theme === themes.light
-                    ? colors.light.white
-                    : colors.dark.maastrichtBlue
-                }
-              />
-            </View>
-          )}
-        </ScrollView>
-      </SafeAreaView>
-    );
-  }
-}
+  return (
+    <SafeAreaView style={[styles.wrapper, styles.theme.wrapper]}>
+      <HeaderBackButton
+        title={'settings.backup_phrase.title'}
+        onPress={navigation.goBack}
+      />
+      <ScrollView style={styles.container}>
+        <PassphraseCopy passphrase={data.recoveryPhrase} />
+        <P style={[styles.QRText, styles.theme.text]}>
+          {t('Private use only')}
+          <A style={styles.button} onPress={toggleQRCode}>
+            &nbsp;{passphraseRevealed ? t('Hide QR code') : t('Show QR code')}
```

Comment:
remove extra space at the beginning (use padding if it necessary
```suggestion
            {passphraseRevealed ? t('Hide QR code') : t('Show QR code')}
```